reference/src/index-syntax.md

```diff
@@ -111,21 +111,21 @@ structure that takes a default value if the index is out of bounds:
 #[syntax(index)]
 public fun borrow_or_set<Key: copy, Value: drop>(
     input: &mut MTable<Key, Value>,
-    key: &Key,
+    key: Key,
     default: Value
 ): &mut Value {
-    if (contains(input, *key)) {
+    if (contains(input, key)) {
```

Comment:
Can we remove copy if we write this as
```suggestion
    if (contains(input, &key)) {
```

Reply:
You could... but it would be annoying generally lol. 

Take a look at the `table` API, all of the usages of `Key` are by-value because we lack life-time annotations. So the result would borrow both from the key and the table itself. The key is problematic particularly because the reference might be local